### PR TITLE
Inject templates for ASR datasets

### PR DESCRIPTION
**Pull request by lewtun**
This PR adds ASR templates for 5 of the most common speech datasets on the Hub, where "common" is defined by the number of models trained on them.

I also fixed a bunch of the tags in the READMEs 😎

### CODE REVIEW

**Comment by lewtun** [comment]
Wait until #2567 is merged so we can benefit from the tagger :)

File: datasets/common_voice/README.md
Comment (lhoestq) [inline datasets/common_voice/README.md:66]: We should keep  zh-CN, zh-HK and zh-TW since they are valid codes according to [BCP-47](https://tools.ietf.org/search/bcp47)

The `Hans` subtag means that the text uses the Simplified Chinese script

Therefore I'd suggest to add these three to our list of supported languages

File: datasets/common_voice/README.md
Comment (lhoestq) [inline datasets/common_voice/README.md:46]: we could add this language to our list of supported languages

File: datasets/common_voice/README.md
Comment (lhoestq) [inline datasets/common_voice/README.md:50]: these are valid BCP-47 codes according to wikipedia, we can add them to our list of languages

**Review by lhoestq — COMMENTED** [review]
Awesome thank you !
Great work for the tags as well

A few comments about some language codes that we are missing in our languages list:

File: datasets/common_voice/README.md
Comment (lewtun) [reply datasets/common_voice/README.md:66]: sounds good, will add them to `datasets`!

**Comment by lewtun** [comment]
thanks for the feedback @lhoestq! i've added the new language codes and this PR should be ready for a merge :)

**Review by lhoestq — APPROVED** [review]
Thanks ^^

**Pull request merged**
